modules/mesh-task/variables.tf

```diff
@@ -136,22 +142,16 @@ variable "outbound_only" {
   default     = false
 }
 
-variable "consul_image" {
-  description = "Consul Docker image."
-  type        = string
-  default     = "public.ecr.aws/hashicorp/consul:1.15.4"
-}
-
 variable "consul_ecs_image" {
   description = "consul-ecs Docker image."
   type        = string
-  default     = "public.ecr.aws/hashicorp/consul-ecs:0.6.0"
+  default     = "hashicorpdev/consul-ecs:latest"
```

Comment:
We'd ideally want to use the `hashicorppreview` version but we are unsure about the new consul ECS version yet. Until then we'll use the dev image